code/datums/mind.dm

```diff
@@ -1395,298 +177,6 @@
-				ticker.mode.red_deity_prophets -= src
-
-			ticker.mode.red_deity_followers |= src
-			current << "<span class='danger'><B>You are now a follower of the red cult's god!</b></span>"
-
-			special_role = "Hand of God: Red Follower"
-			. = 1
-		if("blue")
-			//Remove old allegiances
-			if(src in ticker.mode.red_deity_followers || src in ticker.mode.red_deity_prophets)
-				current << "<span class='danger'><B>You are no longer a member of the Red cult!<B></span>"
-
-			ticker.mode.red_deity_followers -= src
-			ticker.mode.red_deity_prophets -= src
-			current.faction -= "red god"
-			current.faction |= "blue god"
-
-			if(src in ticker.mode.blue_deity_prophets)
-				current << "<span class='danger'><B>You have lost the connection with your deity, but you still believe in their grand design, You are no longer a prophet!</b></span>"
-				ticker.mode.blue_deity_prophets -= src
-
-			ticker.mode.blue_deity_followers |= src
-			current << "<span class='danger'><B>You are now a follower of the blue cult's god!</b></span>"
-
-			special_role = "Hand of God: Blue Follower"
-			. = 1
-		else
-			return 0
-
-	ticker.mode.update_hog_icons_removed(src,"red")
-	ticker.mode.update_hog_icons_removed(src,"blue")
-	//ticker.mode.greet_hog_follower(src,colour)
-	ticker.mode.update_hog_icons_added(src, colour)
-
-/datum/mind/proc/make_Handofgod_prophet(colour)
-	. = 0
-	switch(colour)
-		if("red")
-			//Remove old allegiances
-
-			if(src in ticker.mode.blue_deity_followers || src in ticker.mode.blue_deity_prophets)
-				current << "<span class='danger'><B>You are no longer a member of the Blue cult!<B></span>"
-				current.faction -= "blue god"
-			current.faction |= "red god"
-
-			ticker.mode.blue_deity_followers -= src
-			ticker.mode.blue_deity_prophets -= src
-			ticker.mode.red_deity_followers -= src
-
-			ticker.mode.red_deity_prophets |= src
-			current << "<span class='danger'><B>You are now a prophet of the red cult's god!</b></span>"
-
-			special_role = "Hand of God: Red Prophet"
-			. = 1
-		if("blue")
-			//Remove old allegiances
-
-			if(src in ticker.mode.red_deity_followers || src in ticker.mode.red_deity_prophets)
-				current << "<span class='danger'><B>You are no longer a member of the Red cult!<B></span>"
-				current.faction -= "red god"
-			current.faction |= "blue god"
-
-			ticker.mode.red_deity_followers -= src
-			ticker.mode.red_deity_prophets -= src
-			ticker.mode.blue_deity_followers -= src
-
-			ticker.mode.blue_deity_prophets |= src
-			current << "<span class='danger'><B>You are now a prophet of the blue cult's god!</b></span>"
-
-			special_role = "Hand of God: Blue Prophet"
-			. = 1
-
-		else
-			return 0
-
-	ticker.mode.update_hog_icons_removed(src,"red")
-	ticker.mode.update_hog_icons_removed(src,"blue")
-	ticker.mode.greet_hog_follower(src,colour)
-	ticker.mode.update_hog_icons_added(src, colour)
-
-/datum/mind/proc/make_Handofgod_god(colour)

```

Comment:
Going to be honest here, the fact that hand of god is arbitarily locked to just two colours actually makes the code more complicated than if it was just X colours.